runtime/include/tasks/massivethreads/tasks-massivethreads.h

```diff
@@ -58,7 +58,8 @@ typedef struct {
   c_sublocid_t requestedSubloc;
   chpl_fn_int_t requested_fid;
   chpl_fn_p requested_fn;
-  chpl_task_prvData_t prv;
+  chpl_taskID_t id;
+  //chpl_task_prvData_t prv;
```

Comment:
I'd expect it to include chpl_task_prvData here so you can return it in chpl_task_getPrvData.